builder/dispatchers.go

```diff
@@ -53,7 +53,6 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 		// name  ==> args[j]
 		// value ==> args[j+1]
 		newVar := args[j] + "=" + args[j+1] + ""
-		commitStr += " " + newVar
```

Comment:
Good catch and I have no idea why it was removed - I don't recall being in that code since its about LABEL and not ENV.  Very odd.  It does make me wonder why the tests didn't fail though when I was removed.  I'll work on adding a test to catch that.